### PR TITLE
Form 10-10CG: 2020-10-01 Schema Changes and new PDF

### PR DESCRIPTION
**Pull request by kevinmirc**
Fixes: https://github.com/department-of-veterans-affairs/va.gov-team/issues/13424

**Description**
The Caregiver Program will be publishing a new version of the paper 10-10CG. This PR updates the PDF fill to use the soon to be published PDF.

This PR also supports the 10-10cg schema changes:
- removal of insurance questions
- additional insurance questions
- non required ssns for caregivers

Schema PR: https://github.com/department-of-veterans-affairs/vets-json-schema/pull/505
UI PR: https://github.com/department-of-veterans-affairs/vets-website/pull/14163

Relates To: https://github.com/department-of-veterans-affairs/va.gov-team/issues/13574

### CODE REVIEW

File: Gemfile
Comment (f1337) [inline Gemfile:112]: Do we want to wait until the other PR is merged, before merging this change?

**Review by f1337 — COMMENTED** [review]
Looking good! I made an inline comment re: the vets-json-schema branch. We can also ignore the # lines warning for this one: the code changes are small, the rest is PDF fixtures.

Muppet with sunglasses saying 'Looking good'

File: Gemfile
Comment (kevinmirc) [reply Gemfile:112]: Yes, waiting for the schema to be merged and I can update this to reference master.

**Review by f1337 — APPROVED** [review]
Ship it! :shipit:
Guinea pigs let out of a pen and running like lemmings

**Comment by kevinmirc** [comment]
Blocked by previous breaking change to vets-json-schema: https://github.com/department-of-veterans-affairs/vets-json-schema/pull/513#issuecomment-696907742

**Comment by kevinmirc** [comment]
We can merge after this fix is merged: https://github.com/department-of-veterans-affairs/vets-api/pull/4951

**Pull request merged**
